src/ui.button/configs/default.config.js

```diff
@@ -144,7 +119,7 @@ export default /*tw*/ {
       { square: true, size: "lg", class: "py-4" },
     ],
   },
-  text: "whitespace-nowrap px-1.5",
```

Comment:
Removed paddings from here and added gap to button element. This way we will have gap between slot elements and text only when we actually have any element in slot